src/triage/component/architect/utils.py

```diff
@@ -20,3 +25,44 @@ def feature_list(feature_dictionary):
 
 def str_in_sql(values):
     return ','.join("'{}'".format(value) for value in values)
+
+def retry_if_db_error(exception):
```

Comment:
We should create an issue for merging this with the other version as well (or just include this with ensure_db as something we want to centralize). 

I'm not sure what the right solution is; I think maybe results_schema should end up as something more generic and basically house all of our database code.

Reply:
@thcrock, I realized that actually did not end up using this function, so I deleted it.